Add how to run MeiliSearch in README (#13)
meilisearch/docs-scraper: README.md

A scraper for your documentation website that indexes the scraped content into a MeiliSearch instance.

[MeiliSearch](https://github.com/meilisearch/MeiliSearch) is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box

> This scraper is used in production and crawls the [MeiliSearch documentation](https://docs.meilisearch.com/) on each deployment.

Author: To be honest I wrote that without thinking it could have a difference 😂
I found out this difference: https://www.skylynapps.com/scrapemate/learn/help/general/difference-between-crawler-and-scraper.php
Still not really clear haha
I'm going to update the docs then.

## Installation and Usage

### Run your MeiliSearch Instance

> First of all, you need to run your own MeiliSearch instance. Indeed, the scraper will crawl your website and automatically fill the content into it.<br>

Reviewer: ```suggestion
First of all, you need to run your own MeiliSearch instance. This scraper will crawl your website and automatically index its content in MeiliSearch.<br>
```

MeiliSearch is open-source and can run on your own server! 😄

**Without running a MeiliSearch instance, the scraper will not work.**

Here is the [documentation to install and run MeiliSearch](https://docs.meilisearch.com/guides/advanced_guides/installation.html#installation).

> _A tutorial about how to run MeiliSearch in production is coming..._

Reviewer: I see what you did there 🙈

Author: So much pressure on you now

Reviewer 2: Coming... :)
